spec/models/form/question_spec.rb

```diff
@@ -118,7 +118,7 @@
 
       it "displays a formatted answer label" do
         case_log.mrcdate = Time.zone.local(2021, 10, 11)
-        expect(subject.answer_label(case_log)).to eq("11 Oct 2021")
+        expect(subject.answer_label(case_log)).to eq("11 October 2021")
```

Comment:
Would be good to have tests for Time display as well (particularly since there's a couple of edge cases there (Midnight, Midday ect)

Reply:
Added initializer spec.